modules/blazingsql/cluster.js

```diff
@@ -0,0 +1,75 @@
+const cluster = require('cluster');
+const { BlazingContext, UcpContext } = require('@rapidsai/blazingsql');
+const { Series, DataFrame } = require('@rapidsai/cudf');
+
+const createTable = 'createTable';
+const runQuery = 'runQuery';
+const queryRan = 'queryRan';
+
+const numberOfWorkers = 2;
+
+const ucp_context = new UcpContext();
+
+if (cluster.isMaster) {
```

Comment:
https://nodejs.org/api/child_process.html#child_process_advanced_serialization

```suggestion
if (cluster.isMaster) {
  cluster.setupMaster({ serialization: 'advanced' });
```